nnsvs/model.py

```diff
@@ -58,7 +57,7 @@ def __init__(self, in_dim, hidden_dim, out_dim, num_layers=2, dropout=0.0):
         self.last_linear = nn.Linear(hidden_dim, out_dim)
         self.relu = nn.ReLU()
         self.dropout = nn.Dropout(dropout)
-
+        self.prediction_type="deterministic"
```

Comment:
Same as above.

Reply:
I changed to use  predictin_type() of BaseModel in the latest commit.